sdk/java/build.gradle

```diff
@@ -29,7 +29,7 @@ java {
 
 compileJava {
     options.fork = true
-    options.forkOptions.jvmArgs.addAll(["-Xmx4g"])
+    options.forkOptions.jvmArgs.addAll(["-Xmx16g"])
```

Comment:
Moar memory.

Reply:
yeah, came from the pulumi-java upgrade :)